oxmysql.js

```diff
@@ -46,9 +46,11 @@ const formatQuery = (query, params) => {
     query = query.replace(/@(\w+)/g, (raw, match) => `:${match}`);
 
     let newParams = {};
-    Object.keys(params).forEach((param) => {
-      newParams[param.replace("@", "")] = params[param];
-    });
+    if (!params) {
```

Comment:
Should just be
```js
if (params) {
```

But there's a fix on the way for this stuff anyway.

Reply:
can be closed if there is a solution already coming, thanks